packages/framework/fluid-static/src/rootDataObject.ts

```diff
@@ -46,6 +46,9 @@ class RootDataObject
 {
 	private readonly initialObjectsDirKey = "initial-objects-key";
 	private readonly _initialObjects: LoadableObjectRecord = {};
+	public get IRootDataObject() {
```

Comment:
Typically we'd have a companion interface `IProvideRootDataObject` which has `IRootDataObject: IRootDataObject` and then `IRootDataObject extends IProvideRootDataObject` and returns itself like this.

@anthony-murphy - Just checking that we're not trying to move away from that practice.